primitives/npos-elections/src/reduce.rs

```diff
@@ -506,8 +506,8 @@ fn reduce_all<A: IdentifierT>(assignments: &mut Vec<StakedAssignment<A>>) -> u32
 										};
 
 										if next_value.is_zero() {
-											// if the removed edge is from the current assignment, dis_index
-											// should NOT be increased.
+											// if the removed edge is from the current assignment,
+											// dis_index should NOT be increased.
```

Comment:
Nitpicking but should this be `dis_index`?

Reply:
honestly I am not sure what that is :D I changed it to `index`.